lib/stdlib-extended.nix

```diff
@@ -68,6 +68,8 @@ in
     mkSetLuaBinding = binding: action:
       mkLuaBinding binding.value action binding.description;
 
+    defaultAttributes = attr: self.mapAttrs (name: value: self.mkDefault value) attr;
```

Comment:
I'm not sure if I like `defaultAttributes` as the function name. It should hint towards either the fact that it's a label (e.g. defaultLabels) or the fact that it's a constructor function (e.g mkDefaultLabelAttr)